.python-version

```diff
@@ -0,0 +1 @@
+3.9.6
```

Comment:
is this strictly necessary? could it be addressed in a separate hygiene PR?

Reply:
Moved this to https://github.com/python/pythondotorg/pull/2114